SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Channel/Channel.cs

```diff
@@ -43,7 +43,7 @@ public sealed class Channel : IChannelUnit
             _batchTimeInterval = batchTimeInterval;
             _batchScheduled = false;
             _enabled = true;
-            DeviceInformationHelper.InformationInvalidated += (sender, e) => InvalidateDeviceCache();
+            //DeviceInformationHelper.InformationInvalidated += (sender, e) => InvalidateDeviceCache();
```

Comment:
Let's maintain Microsoft-specific code as-is.
In case of modifications required we could use #if TIZEN macro defined in Tizen-specific DLL projects.